src/types.rs

```diff
@@ -14,6 +14,9 @@ pub enum PlatformType {
     SunOs,
     Windows,
     Android,
+    FreeBSD,
+    NetBSD,
+    OpenBSD,
```

Comment:
One final thing: As per https://github.com/rust-lang/api-guidelines/blob/385ba009b19672a373cc65c98368527d2de82482/src/naming.md#casing-conforms-to-rfc-430-c-case , these should be `FreeBsd`, `NetBsd`, and `OpenBsd` instead. Other than that, we are good to go here

Reply:
Noted, will fix it in a couple of minutes.